### DISCUSSION TITLE
Please add translation file of Japanese

### DISCUSSION THREAD

**Discussion by vv4ta783** [sandboxie-plus/Sandboxie#2721] [Documents and notes]
Hi, everyone!
I want to translate sandboxie-plus.
But I didn't know how to make .ts file.
Please make Japanese translation file for me.
Thenk you!

**Comment by okrc** [comment]
For localization instructions and tips, see #1123
I created the ts file under [my branch](https://github.com/okrc/Sandboxie/tree/translation_ja) and you can use it to create a new branch.
Other parts you still need to pay attention to, you can use as a reference https://github.com/sandboxie-plus/Sandboxie/pull/2438#issuecomment-1305548450

**Comment by vv4ta783** [comment]
I'll reference it again.
Thank you for @okrc !!
